gpt4all-chat/mysettings.cpp

```diff
@@ -122,6 +122,7 @@ static QString getUiLanguage(const QString directory, const QString fileName)
 static QStringList getUiLanguages(const QString &modelPath)
 {
     QStringList languageList;
+    QStringList releasedLanguages = { "en_US", "it_IT", "zh_CN", "zh_TW" };
```

Comment:
releasedLanguages should be static const. We could avoid the runtime constructor if we used std::ranges::find with a C-style array or std::array, but that's not a big deal.
```suggestion
    static const QStringList releasedLanguages { "en_US", "it_IT", "zh_CN", "zh_TW" };
    QStringList languageList;
```